src/SplitPackages/Frameworks.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Frameworks;
+
+namespace SplitPackages
+{
+    public static class Frameworks
+    {
+        public static string NetStandardApp15 => FrameworkConstants.CommonFrameworks.NetStandardApp15.DotNetFrameworkName;
+        public static string Net451 => FrameworkConstants.CommonFrameworks.Net451.DotNetFrameworkName;
+        public static string DnxCore50 => FrameworkConstants.CommonFrameworks.DnxCore50.DotNetFrameworkName;
```

Comment:
Do we have any projects targeting this as yet? We could do a quick pass to remove this entirely (or is this just an import for old code?)